src/composables/PluginManager.js

```diff
@@ -0,0 +1,149 @@
+import { readTextFile } from 'src/composables/Files';
+import plugins from 'src/plugins';
+
+let instanciatePlugins = [];
+
+/**
+ * Initialize plugin metadata and definitions.
+ *
+ * @param {Plugin} plugin - instantiate plugin.
```

Comment:
replace `{Plugin}` by `{Object}`, but thank's for the idea, we will improve the plugin-core with a plugin class :)

Reply:
Done